conf/distro/include/openxt-xen.inc

```diff
@@ -0,0 +1,5 @@
+# xen version and source archive
+XEN_VERSION="4.6.1"
```

Comment:
That should be upgraded to 4.6.4, to match the upgraded version in master before it can be merged.